runtime/src/main/java/com/datastax/oss/quarkus/runtime/driver/internal/DefaultMutinyGraphReactiveResultSet.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.driver.internal;
+
+import com.datastax.dse.driver.api.core.graph.reactive.ReactiveGraphNode;
+import com.datastax.dse.driver.api.core.graph.reactive.ReactiveGraphResultSet;
+import com.datastax.oss.driver.api.core.cql.ExecutionInfo;
+import com.datastax.oss.quarkus.runtime.driver.api.MutinyGraphReactiveResultSet;
+import com.datastax.oss.quarkus.runtime.reactive.Wrappers;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.groups.MultiBroadcast;
+import io.smallrye.mutiny.groups.MultiCollect;
+import io.smallrye.mutiny.groups.MultiConvert;
+import io.smallrye.mutiny.groups.MultiGroup;
+import io.smallrye.mutiny.groups.MultiOnCompletion;
+import io.smallrye.mutiny.groups.MultiOnEvent;
+import io.smallrye.mutiny.groups.MultiOnFailure;
+import io.smallrye.mutiny.groups.MultiOnItem;
+import io.smallrye.mutiny.groups.MultiOverflow;
+import io.smallrye.mutiny.groups.MultiSubscribe;
+import io.smallrye.mutiny.groups.MultiTransform;
+import java.util.concurrent.Executor;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import org.reactivestreams.Subscriber;
+
+public class DefaultMutinyGraphReactiveResultSet implements MutinyGraphReactiveResultSet {
+
+  private final Multi<ReactiveGraphNode> multi;
+  private final Multi<ExecutionInfo> executionInfos;
+
+  public DefaultMutinyGraphReactiveResultSet(ReactiveGraphResultSet reactiveGraphResultSet) {
+    multi = Wrappers.toMulti(reactiveGraphResultSet);
+    @SuppressWarnings("unchecked")
+    Multi<ExecutionInfo> executionInfos =
```

Comment:
To explain this: the original API returns `Publisher<? extends XYZ>`. But the wildcard was a mistake, the best signature would be `Publisher<XYZ>`. So I figured that we wouldn't propagate the mistake to this new API, so `MutinyReactiveQueryMetadata` has `Multi<XYZ>` not `Multi<? extends XYZ>`. But at some point we need an unchecked cast to go from the driver rx API to the extension's rx API.

Reply:
If we can reduce the code by this cast I am +1 for it